kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue

```diff
@@ -406,7 +406,11 @@
           SELECT_QUIZ: PageNames.QUIZ_SELECT_PRACTICE_QUIZ,
         }[value];
         nextRoute.name = nextRouteName;
-        this.$router.push(nextRoute);
+        if (nextRoute.name === PageNames.EXAM_CREATION_ROOT) {
+          this.$router.push(nextRoute);
+        } else {
+          this.$router.replace(nextRoute);
+        }
```

Comment:
@AllanOXDi I think we should remove this and test as I don't think it really helps us.